src/components/Promise/promise.css

```diff
@@ -11,7 +11,7 @@
 }
 .lil-promise-horizontal .lil-promise-content {
     height: 75%;
-    @apply lil-w-full lil-break-words lil-mt-4;
+    @apply lil-w-full lil-break-words lil-pt-6;
```

Comment:
verified for all promises ?

Reply:
Yep, only adds the spacing between image and content.. checked with your PR too, this change will be consistent